_utils/terror_gen/checker_template.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/pingcap/dm/pkg/terror"
+)
+
+const (
+	developErrorFile     = "errors_develop.txt"
+	releaseErrorFile     = "errors_release.txt"
+	generatedCheckerFile = "checker_generated.go"
```

Comment:
define `generatedCheckerFile` both in _checker_template.go_ and _gen.go_.

Reply:
These files are designed as assist tools, so they are placed in a directory with underscore leading. Some duplicate definition may be acceptable. any better suggestions?

Reply:
can we remove the one in _gen.go_? no better suggestion.

Reply:
how can `gen.go` get this value without code level import?

Reply:
generate this value from `gen.go`